great_expectations/data_context/data_context.py

```diff
@@ -1979,17 +1982,16 @@ def get_config(
         if mode == "typed":
             return config
 
-        elif mode == "commented_map":
+        if mode == "commented_map":
             return config.commented_map
```

Comment:
This further supports the idea of an enum in my opinion. If we're narrowing down certain values in multiple places, I think we should leverage enums and find a way to keep the user interface the same (`mode="yaml"`)

Reply:
@cdkini Agreed -- in the next PR -- and for all occurrences, including `DataContextBase`.